marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserArtifactsCommand.java

```diff
@@ -101,7 +98,7 @@ private PropertiesModuleManager getModulesManager() {
                 }
             }
         }
-
+        pmm.initialize();
```

Comment:
This is a critical change - can you add a comment here to explain why this is being done, which I believe is so that in the QuickStart context, the command only loads what needs to be loaded when the user makes a change?